css/style.css

```diff
@@ -17,3 +18,251 @@ body {
  */
 
 
+
+/* HEADER */
+
+.wrapper {
+    width: 1170px;
+    margin: 0 auto;
+}
+
+
+header{
+    height: 50px;
+    background: whitesmoke;
+    width: 100%;
+    position:relative;
```

Comment:
Adding a `padding-top` to this class helps the menu buttons to shift down and be closer what the design is asking you.

Reply:
When I looked at the image - the nave bar was closer to the top of the screen than the centre of the box.
I would have changed it otherwise - but thank you for pointing that out; I must have miss-seen.